src/ratbag_emu/protocol/util/profile.py

```diff
@@ -46,3 +52,9 @@ def dpi(self):
     @dpi.setter
     def dpi(self, dpi):
         self.x_dpi = self.y_dpi = dpi
+
+    def get_dpi(self):
```

Comment:
python has decorators for getters/setters that can be helpful:
```python
class A(object):
  def __init__(self, value):
    self._value = value

  @property
  def value(self):
    return self._value

  @value.setter
  def value(self, new_value):
    assert value > 0
    self._value = value
```

And then you can use it with:

```python
a = A(5)
print(a.value)
a.value = 10
a.value = -6  # <- this should raise an assert
```

Reply:
I am aware and I am using that above. Admittedly it's not the best code I've written but `get_dpi()` is just a function to quickly get the active dpi value.